modules/notifications/notifications.php

```diff
@@ -217,19 +193,33 @@ public function get_notification_footer( $post ) {
 	 * @param string $message_headers. (optional) Message headers
 	 */
 	public function send_email( $action, $post, $subject, $message, $message_headers = '' ) {
+		// Do nothing if the post isn't set.
+		$post_id = $post->ID;
```

Comment:
I'm not sure how this would happen, but it looks like you added it so there probably was an issue. I'd double check with an `isset()` just to avoid potential warnings:

```suggestion
		$post_id = isset( $post->ID ) ? $post->ID : false;
```

Reply:
This was actually something that was taken from Edit Flow, so I left it in there. I was actually leaning towards just removing it tbh considering how this code is triggered in the first place